release/train_tests/benchmark/recsys/torchrec_runner.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+
+import torch
+from torchrec.distributed import TrainPipelineSparseDist
+from torchrec.optim.keyed import CombinedOptimizer, KeyedOptimizerWrapper
+from torchrec.optim.optimizers import in_backward_optimizer_filter
+
+import ray.train
+import ray.train.torch
+
+from runner import TrainLoopRunner
+
+
+logger = logging.getLogger(__name__)
+
+
+class TorchRecRunner(TrainLoopRunner):
+    def setup(self):
+        device = ray.train.torch.get_device()
+
+        self.model = self.factory.get_model()
+
+        dense_optimizer = KeyedOptimizerWrapper(
+            dict(in_backward_optimizer_filter(self.model.named_parameters())),
+            lambda params: torch.optim.Adagrad(params, lr=15.0, eps=1e-8),
+        )
+        self.optimizer = CombinedOptimizer(
+            [self.model.fused_optimizer, dense_optimizer]
+        )
+        # lr_scheduler = LRPolicyScheduler(
+        #     optimizer, args.lr_warmup_steps, args.lr_decay_start, args.lr_decay_steps
+        # )
+
+        self.pipeline = TrainPipelineSparseDist(
+            self.model, self.optimizer, device, execute_all_batches=True
+        )
+
+    def train_step(self, train_dataloader):
+        self.pipeline.progress(train_dataloader)
+
+    def validate_step(self, val_dataloader):
+        return 0
+
+    def _save_training_state(self, local_dir: str):
+        return
```

Comment:
why early return here? 
Can you add a comment if the following is not needed?

Reply:
Oh oops, I will add in the model saving logic.